anchor/programs/pubkey-protocol/src/instructions/profile/profile_create.rs

```diff
@@ -65,6 +65,7 @@ pub fn profile_create(ctx: Context<ProfileCreate>, args: ProfileCreateArgs) -> R
         username,
         name,
         avatar_url,
+        bio,
```

Comment:
We don't need the bio in the `profile_create` instruction, only in the `profile_update` one.